config/nuttx/chip-gn/toolchain/BUILD.gn

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
I realize part of this is copy & paste, however likely most of this file is likely completely new. Does 2020 really apply or is this 2024?